.github/actions/publish/action.yml

```diff
@@ -39,9 +39,8 @@ runs:
         args: release ${{ inputs.dry-run == 'true' && '--skip=publish' || '' }}
       env:
         GITHUB_TOKEN: ${{ inputs.token }}
-        HOMEBREW_DEPLOY_KEY: ${{ inputs.homebrew-gh-secret }}
```

Comment:
did we mean to remove the `HOMEBREW_DEPLOY_KEY`?

Reply:
Nope. Probably a weird merge.

Reply:
Oh thanks for restoring that